gen/definitions/network_access_authentication_rule_update_rank.yaml

```diff
@@ -0,0 +1,67 @@
+---
+name: Network Access Authentication Rule Update Rank
+# Manual update in Create function in resource file to add AuthRuleId to path
+# and populate attributes from existing resource using GET request in Create and Update functions
+res_description:
+  'This resource is used to update rank field in network access authentication rule. It serves as a workaround for the 
+  ISE API/Backend limitation which restricts rank assignments to a strictly incremental sequence.
+  By utilizing this resource and network_access_authentication_rule resource, you can bypass the APIs limitation.
+  Creation of this resource is performing PUT operation (Update) and it only tracks rank field.
+  When this resource is destroyed, no action is performed on ISE and resource is just removed from state.'
+rest_endpoint: /api/v1/policy/network-access/policy-set/%v/authentication
+data_source_name_query: true
+put_create: true
+no_delete: true
+no_import: true
+no_data_source: true
+skip_minimum_test: true
+ignore_delete_error: Attempted to delete default
```

Comment:
As we don't have delete, this can be removed.

Reply:
done